Skills.md

```diff
@@ -4,11 +4,11 @@ https://api.careerbuilder.com/core/tagging/skills
 
 This API supports the HTTP/GET and HTTP/POST methods.  
 
-This API currently supports version 4.1 only. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.1 contains the latest taxonomy and is more accurate and has a few changes of field names and error formats to conform to standards.
+This API currently supports versions 4.1 and 4.2. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.2 contains the latest taxonomy and is more accurate and has a few changes of field names and error formats to conform to standards.

```

Comment:
The "conform to standards" bit is legacy from v4.0 and below which were non-API standards-compliant. I would remove that half of the sentence and just summarize the 4.1->4.2 changes